crates/wasm-shrink/tests/tests.rs

```diff
@@ -44,8 +44,10 @@ fn shrink_to_empty_allowed() -> Result<()> {
 #[test]
 fn smoke_test() -> Result<()> {
     let info = WasmShrink::default().attempts(100).run(wasm(), |wasm| {
-        let wat = wasmprinter::print_bytes(&wasm)?;
-        Ok(wat.contains("local.get"))
+        match wasmprinter::print_bytes(&wasm) {
+            Ok(wat) => Ok(wat.contains("local.get")),
+            Err(_) => Ok(false),
+        }
     })?;
```

Comment:
This change technically shouldn't be required, `wasm-mutate`, when given a valid module, should always produce a valid module. What you've run into here though was a preexisting bug which I've fixed at https://github.com/bytecodealliance/wasm-tools/pull/2135, so I'll try to remember to ensure this gets reverted back depending on the order things land

Reply:
Ah ok with #2135 now merged if you rebase this PR then this change should no longer be necessary

Reply:
Done!